packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -228,17 +235,15 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
             "OptimismPortal: you cannot send messages to the portal contract"
         );
 
-        // Get the output root.
+        // Get the output root. This will fail if there is no
+        // output root for the given block number.
         L2OutputOracle.OutputProposal memory proposal = L2_ORACLE.getL2Output(_l2BlockNumber);
```

Comment:
Argument for doing the math in L2OutputOracle is because otherwise this will fail if the user doesn't provide the exact correct block number